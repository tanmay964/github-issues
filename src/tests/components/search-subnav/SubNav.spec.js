import React from 'react';
import renderer from 'react-test-renderer';
import SubNav from '../../../components/search-subnav/SubNav';

it('renders correctly', () => {
  const tree = renderer
    .create(<SubNav />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
