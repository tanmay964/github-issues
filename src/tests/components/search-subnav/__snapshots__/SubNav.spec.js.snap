// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="sc-dnqmqq jDUlnv"
>
  <div
    className="sc-gZMcBi eXUiWO"
  >
    <div>
      <div
        className="sc-gzVnrw xOIAi"
      >
        Filters
        <span
          className="sc-EHOje ifVNKl"
        />
      </div>
      <div
        className="sc-iwsKbI gyOusD"
      >
        <input
          className="sc-bZQynM hclrXZ"
          defaultValue="is:issue is:open "
          type="text"
        />
        <svg
          aria-hidden="true"
          className="sc-bxivhb sc-ifAKCX iNGdBl"
          height="16"
          width="16"
        >
          <path
            d="M15.7 13.3l-3.81-3.83A5.93 5.93 0 0 0 13 6c0-3.31-2.69-6-6-6S1 2.69 1 6s2.69 6 6 6c1.3 0 2.48-.41 3.47-1.11l3.83 3.81c.19.2.45.3.7.3.25 0 .52-.09.7-.3a.996.996 0 0 0 0-1.41v.01zM7 10.7c-2.59 0-4.7-2.11-4.7-4.7 0-2.59 2.11-4.7 4.7-4.7 2.59 0 4.7 2.11 4.7 4.7 0 2.59-2.11 4.7-4.7 4.7z"
            fillRule="evenodd"
          />
        </svg>
      </div>
    </div>
    <div
      className="sc-bdVaJa ketRLB"
    >
      <a
        className="sc-bwzfXH OhcHd"
        href="https://github.com/facebook/react/labels"
      >
        Labels
      </a>
      <a
        className="sc-htpNat ldolpf"
        href="https://github.com/facebook/react/milestones"
      >
        Milestones
      </a>
    </div>
  </div>
  <a
    className="sc-htoDjs hFrhTM"
    href="https://github.com/facebook/react/issues/new"
  >
    New issue
  </a>
</div>
`;
